code/exp/util.py

```diff
@@ -54,7 +57,7 @@ def minibatch_woodbury_update(X, H_inv):
     """
     batch_size = tf.shape(X)[0]
 
-    M0 = tf.eye(batch_size) + tf.matmul(X, tf.matmul(H_inv, X, transpose_b=True))
```

Comment:
I changed dtype for tf.eye to float64 for compatibility reasons. All other changes in util.py are mostly style changes.